src/modules/information/service/information.service.ts

```diff
@@ -0,0 +1,59 @@
+import { prisma } from '@lib/prisma/prisma';
+import { UpdateInformationProps } from '../types';
+
+export default class InformationService {
+  static updateInformationByUserId = async (
+    id: string,
+    body: UpdateInformationProps,
+  ) => {
+    const user = await prisma.user.findUnique({
+      where: { id },
+    });
+    if (!user) {
+      throw new Error('User not found');
+    }
+
+    const information = await prisma.information.findFirst({
+      where: { userId: id },
+    });
+
+    const informationUpdated = information
+      ? await prisma.information.update({
```

Comment:
Have you try `upsert` https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#upsert

Reply:
oh cool, idk yet